docs/manual/creating-a-checker.tex

```diff
@@ -144,13 +144,16 @@
 \begin{Verbatim}
 myPackage/
   | qual/                               type qualifiers
-  | MyPropChecker.java                  [optional] interface to the compiler
+  | MyPropChecker.java                  interface to the compiler
   | MyPropVisitor.java                  [optional] type rules
   | MyPropAnnotatedTypeFactory.java     [optional] type introduction and dataflow rules
 \end{Verbatim}
 %
-Note that \<MyPropChecker.java> is required unless you are building on the
-Subtyping Checker.
+\<MyPropChecker.java> is occasionally optional, such as if you are building
+on the Subtyping Checker.  You may wish to put the \<qual/> directory in a
+separate Maven/Gradle project, if you want to create an artifacts
```

Comment:
`an artifacts` => `an artifact`.  

The Maven term is "module" and the Gradle term is "subproject", but I think what you have written here makes sense.